packages/opentelemetry-context-zone-peer-dep/src/util.ts

```diff
@@ -17,10 +17,11 @@
 import { TargetWithEvents } from './types';
 
 /**
- * check if an object has addEventListener and removeEventListener functions then it will return true
- * @param obj
+ * check if an object has addEventListener and removeEventListener functions then it will return true.
+ * Generally only called with a `TargetWithEvents` but may be called with an unknown / any.
+ * @param obj - The object to check.
  */
-export function isListenerObject(obj: TargetWithEvents = {}): boolean {
+export function isListenerObject(obj: TargetWithEvents | any = {}): boolean {
```

Comment:
Other options considered and open to
- Change the obj type to just any `function isListenerObject(obj: any = {})`
- Add a generic type with a constraint of extending TargetWithEvents, but this requires the caller to cast as any
- Change the type checks to use "named" `obj["addEventListener"]` lookup as (currently) TypeScript doesn't complain on this type of lookup
- Cast each `obj` as any for usage `(obj as any).addEventListener` (just not pretty)